algebra-testing/src/test/java/com/hubspot/assertj/algebra/api/ResultAssertTest.java

```diff
@@ -1,12 +1,11 @@
 package com.hubspot.assertj.algebra.api;
 
-import static com.hubspot.assertj.algebra.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
-
+import com.hubspot.algebra.Result;
 import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
 import org.junit.Test;
 
-import com.hubspot.algebra.Result;
+import static com.hubspot.assertj.algebra.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
```

Comment:
I think your import style settings were messed up, the original order was correct according to our typical style.